### PR TITLE
Monday

### PR DESCRIPTION
**Pull request by evildmp**
Added schedule-related information:
- information about Monday
- "What on earth are code sprints?" page

### CODE REVIEW

File: content/what_are_sprints.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## But I’m not actually a very good Python programmer.
+That’s OK too, neither am I and neither is everyone else. You truly don’t need to be very good Python programmer. In fact, you barely need to be a programmer at all, because even the most novice sprinter can make a very valuable contribution by helping improve documentation, for example.
+
+## I’d be worried about holding up everyone else though.
+It simply doesn’t work like that. For one thing, one of the main points of a sprint is to encourage and help new people. You’ll be able to work at your own pace, learning the things you need to learn as you go along, and you’ll be supported by other people.
+
+## Do I have to stay for a whole sprint?
+Not at all. Drop in just for a bit, if you just want to see what’s like. There'll be plenty of other things on the code day too, such as [clinics and workshops](/code), so don’t miss them. Other people will be dropping in and out of sprints as well.
```
Comment (drvinceknight) [inline content/what_are_sprints.md]: [Typo:] what it's like

**Comment by evildmp** [comment]
Thanks @drvinceknight, fixed.

File: content/code_day.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+type: page
+slug: code
```
Comment (inglesp) [inline content/code_day.md]: Could the slug be `code-day` please?  `code` is too generic.

File: content/code_day.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+title: Code day
+callout_big_1: Our day of workshops, sprints and collaboration
+callout_small: Monday 19th September
+---
+
+The final day of PyCon UK will be dedicated to *Code and collaboration*.
```
Comment (inglesp) [inline content/code_day.md:8]: Does "Code" need a capital "C"?

File: content/code_day.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+These sessions will cover all kinds of things - particular packages, programming disciplines, techniques. In previous
+years we've had workshops on Django, automated testing, how to contribute to open-source projects, mathematics,
+robotics and much, much more.
+
+Would you like to run a workshop? [Tell us about it!](/cfp)
+
+## Sprints
+
+Code sprints provide members of an open-source software community to collaborate in person to tackle outstanding issues
```
Comment (inglesp) [inline content/code_day.md]: I think you're missing a word.  Should it be "opportunities to collaborate"?

File: content/code_day.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+Newcomers are welcome at sprints - so don't think they're only for so-called 'experts'.
+
+You don't need to plan a sprint in advance, but if you [tell us about your sprint](/cfp) we can help advertise it for
+you.
+
+If you've got this far but still aren't sure what a sprint is, you should read [What on earth is a code sprint?](/what-are-sprints).
+
+## Clinics
+
+Take your sickly code to the friendly PyCon code doctor. Take your CV to a recruitment expert.
```
Comment (inglesp) [inline content/code_day.md]: s/PyCon/PyCon UK/

File: content/code_day.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+Would you like to perform a valuable PyCon service by running or taking part in a clinic? [Tell us what you'd like to do.](/cfp)
```
Comment (inglesp) [inline content/code_day.md]: s/PyCon/PyCon UK/

File: content/code_day.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+
+## Other collaboration
+
+Do you have a big idea, that needs some other fine minds to help get it off the ground?
```
Comment (inglesp) [inline content/code_day.md:43]: There's a stray comma here.

File: content/what_are_sprints.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+## So can I join in?
+Absolutely. Everyone’s welcome. It’s not an elite club, or for experts only. All of the projects that will be running sprints welcome anyone who wants to help.
+
+## I’ve never taken part in a sprint before though.
+That’s no problem. There’ll be an introduction to explain how it works, and other people will be only too pleased to help you get started or to work with you.
+
+## What do I need?
+You'll need to know the basics of using the appropriate version control system (it'll probably be Git/GitHub) to checkout code and make pull requests, to know how to use the issue tracker, and so on. You’ll also need your own computer with a suitable Python environment set up on it - but there'll be people to help with that too.
```
Comment (inglesp) [inline content/what_are_sprints.md]: s/checkout/check out/

File: content/code_day.md (hunk 3 above)
Comment (evildmp) [inline content/code_day.md:8]: Same as _Introduction_ on the other page.

File: content/code_day.md (hunk 7 above)
Comment (evildmp) [inline content/code_day.md:43]: Intended as-is.

Comment (inglesp) [inline content/code_day.md:43]: The comma means it doesn't scan well when I read it, but it's only a comma!

**Pull request merged**